apps/volontulo/templates/regulations.html

```diff
@@ -0,0 +1,15 @@
+{% extends "common/col1.html" %}
+{% load main_image %}
+
+{% block title %}Volontulo - podejmij pracę jako wolontariusz{% endblock %}
+
+{% block name %}newsletter-signup{% endblock %}
+
+{% block content-heading %}
+    <img class="img-responsive center-block" src="http://placehold.it/1260x260?text=Albo+jeden+duży+banner" />
+{% endblock %}
+
+{% block content %}
+    <h2>Zapisz się do newslettera</h2>
```

Comment:
the page filename is regulations.html, but you put here newsletter content. It should probably belong in newsletter_signup.html and vice versa